### ISSUE TITLE
[OBJECT DETECTION] Model Training Requirements

### ISSUE THREAD

**Issue by ishaanchowdhary** [ishaanchowdhary/ece4191-s2-2025-r21#3]
As per object detection requirements, we require a functional object detection model to identify and classify different animals.

Required tasks:

1.  Select pretrained object detection model for training
2.  Collect image data for testing camera as well as electrical team camera (Pi Cam v3 NoIR)
3.  Label and classify dataset for both image sets on Roboflow
4.  Train object detection model (one for each camera)
5.  Test performance in real-time
6.  Integrate processing workflow into system and fine-tune processing
